### DISCUSSION TITLE
my colors are not loaded properly.

### DISCUSSION THREAD

**Discussion by ajayd-san** [olimorris/onedarkpro.nvim#192] [Q&A]
hey, I just installed the extension and it does not look alike to vscode. all my variables are read (both on lhs and rhs). And there is  even the `self` keyword is highlighted in red too. 
![neovim](https://github.com/olimorris/onedarkpro.nvim/assets/54715852/bb2df1a3-82c9-4086-96c1-370cadfe919d)
![vscode ](https://github.com/olimorris/onedarkpro.nvim/assets/54715852/6cbd6645-f4b0-4ad2-b902-6ac2cb927a7e)

how do I fix this? Any help is appreciated .

**Comment by olimorris** [comment]
I suggest trying with the minimal config in the bug report template.

Difficult to unpick anyone's config without an example of their setup.
